controllers/grid/pubIds/form/PKPAssignPublicIdentifiersForm.inc.php

```diff
@@ -66,6 +73,7 @@ function fetch($request) {
 		$templateMgr->assign('pubObject', $this->getPubObject());
 		$templateMgr->assign('approval', $this->getApproval());
 		$templateMgr->assign('confirmationText', $this->getConfirmationText());
+		$templateMgr->assign('formParams', $this->getFormParams());
```

Comment:
This appears to be an OJS-only concern (e.g. it's never used in the pkp-lib templates). Is it possible to keep it in OJS only?